android/app/webviewAssets.gradle

```diff
@@ -7,6 +10,45 @@
  * more information.
  */
 
+/**
```

Comment:
Nit: This is formatted to make it be javadoc for `normalizePath`, but it's really not documentation for either of these helpers (it doesn't describe their interface).

Writing a plain `/*` would be a sufficient fix.